### DISCUSSION TITLE
Obfuscate or encrypt json snippet database

### DISCUSSION THREAD

**Discussion by csgabe** [massCodeIO/massCode#328] [Ideas]
You should use BSON( Bin­ary JSON) for snippet database 
https://bsonspec.org/
or encrypt it by using JEF(JSON Encryption Format):
https://cyberphone.github.io/doc/security/jef.html
or this([electron-data-holder](https://www.npmjs.com/package/electron-data-holder)):
https://dev.to/gugocharade/storing-data-in-electron-1gd4

**Comment by antonreshetov** [comment]
Should?

**Reply by csgabe** [reply]
Sorry, I meant you could.
Or easiest would be XOR encryption of database file.
